MAUI/Cartesian-Charts/Plot-Area-Background-View.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Plot Area Background View  in .NET MAUI Chart control | Syncfusion

```

Comment:
Appearance in .NET MAUI Chart control | Syncfusion

Reply:
I have changed the title content.